pkg/kernel/kernel-config/kernel_config-5.10.x-aarch64

```diff
@@ -6462,7 +6465,15 @@ CONFIG_STM_SOURCE_CONSOLE=y
 
 # CONFIG_FPGA is not set
 # CONFIG_FSI is not set
-# CONFIG_TEE is not set
+CONFIG_TEE=m
```

Comment:
Not sure why this is here. We should removal unrelated changes from this PR.

Reply:
Yes, already on it.